### DISCUSSION TITLE
Error Message "Unknown or unsupported novelWriter project file format."

### DISCUSSION THREAD

**Discussion by Reproog** [vkbo/novelWriter#1332] [Help] [answered]
I get this error when I open an existing project (saved in 2.0.4)

"Unknown or unsupported novelWriter project file format. The project cannot be opened with this version of novelWriter. The file was saved with novelWriter version 2.0.2."

I have reinstalled novelWriter via pip, version 2.0.4. 
When I list all my installed packages, there is novelwriter, version 2.0.4. But when I try to open the file nwProject.nwx, Novelwriter tries to open the project with version 1.6, although this version should not exist on my PC.

What can I do to fix this problem?

**Comment by vkbo** [comment, marked as answer]
You have two versions installed somehow. The 1.6 version cannot read a project saved with 2.0.x because the file format changed. You need to uninstall the version 1.6. The pip version will not be launched when you open a nwProject.nwx file directly, only the system installed version. The 1.6 version is probably installed from a different source. What operating system are you on?

If you start the pip version up, you should be able to open the project.

(I converted this to a help topic as this is not a bug.)

**Comment by Reproog** [comment]
I work with Windows 11.

I found an older version on my second hard drive. 
This version has overwritten this newer version. 
I deleted the older version. Now it works. 

Thanks ;)

**Reply by vkbo** [reply]
Great.

Also, please use the reply box when replying to an answer. You just added a second answer to your own question now :)
